### ISSUE TITLE
[BITV] 9.2.4.3/3.9 - Toolbar focus handling

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/text#3863]
- [ ] Fix
- [ ] Backport

Note: When style is applied to selected text in the editing region (div with contenteditable=true) by activating the button for that style the focus is set to the selected text. There are pros and cons for moving the focus. 

In any case the controls in the div with role=menubar (role=toolbar would be more appropriate) should have ONE focus stop for the toolbar, implement arrow navigation for its elements, and support aria-activedescendant and scripted focus management so the focus can be set to the currently active style option (the last used of them when several styles have been applied) to facilitate editing for keyboard users. (2)

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222250672-8c463058-775b-4c0a-b079-830ff7168593.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v3-n9

</details>

- [ ] 9.2.4.3/10.9 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v10-n9

**Comment by datenangebot** [comment]
Can confirm, no arrow navigation, but tabs work fine.

If you click on the expandable menu, the arrows works.
So atm it is mixed. 🙄
